.buildkite/pipeline.yml

```diff
@@ -1,6 +1,7 @@
 env:
   SETUP_GVM_VERSION: 'v0.5.2' # https://github.com/andrewkroh/gvm/issues/44#issuecomment-1013231151
   ELASTIC_PACKAGE_COMPOSE_DISABLE_VERBOSE_OUTPUT: "true"
+  ELASTIC_PACKAGE_MAXIMUM_NUMBER_PARALLEL_TESTS: 3
```

Comment:
If package has enabled system tests in parallel, those tests will be triggered using 3 routines as maximum.